README.md

```diff
@@ -127,6 +127,7 @@ If your service doesn't have Welsh translations you can disable them setting the
       "lookupPageLabels": {
         "title": "Find address",
         "heading": "Find address",
+        "afterHeadingText": "foo",
```

Comment:
Replace with better description?

Reply:
It's an identical pattern to an optional text that is normally left blank.  See "headingWithPostcode" in the config section for selectPageLabels above.  The description for the field is below, and I think it's left this way so people don't copy/paste the config and it shows up clearly on the page as something you need to change.  Happy to change though, will get a proper steer from the ALF team.